cert-expiry-finder

```diff
@@ -294,10 +294,29 @@ expiry_after() {
     esac
 }
 
+list_k8s_cert_expiry_days() {
+    local name namespace notafter unixtime delta context
+    if ! which kubectl; then
+        return
+    fi
+    kubectl config get-contexts --no-headers -oname | while read context
+    do
+        kubectl get cert --context $context -A -ojson \
+            | jq -r '.items[] | (.metadata.name + " " + .metadata.namespace + " " + .status.notAfter)' \
+            | while read name namespace notafter
+        do
+            unixtime=$(date --date="$notafter" +%s)
+            delta=$(((unixtime - global_t0) / 86400))
+            printf '%-7d Kubernetes (NS = %s, N = %s, C = %s)\n' $delta "$namespace" "$name" "$context"
```

Comment:
Please think about the 79 column limit.

Reply:
Also, if we're making up letters/abbreviations, I'd prefer: `NS` `CRT` `CTX`

Reply:
changed